CRM/Import/Parser.php

```diff
@@ -960,9 +945,13 @@ protected function checkContactDuplicate(&$formatValues) {
    * @param string $contactType
    *
    * @return string
+   * @throws \CRM_Core_Exception
    */
   protected function getDefaultRuleForContactType(string $contactType): string {
-    return $contactType . '.Unsupervised';
+    return DedupeRuleGroup::get(FALSE)
```

Comment:
This turns out to be how it works - it there is only ever one Unsupervised rule per type -but the name might NOT be `IndividualUnsupervised` based on my testing of configuring in the UI